docs/how-to/configure.md

```diff
@@ -98,6 +98,15 @@ The checks take 24 hours to complete.
 
 The results can be viewed in *PMM > PMM Database Checks*.
 
+**DBaaS**
+
+Shows whether DBaaS features are activated on this server.
+
+**Public Address**
+
+Public address for accessing this server.
```

Comment:
"now used only for Monitoring needs of DBaaS functionality".

Let's make it clear because it's not used for other features as for now